docs/docs/05-reusable-components.md

```diff
@@ -70,9 +70,26 @@ React.createClass({
     // won't work inside `oneOfType`.
     customProp: function(props, propName, componentName) {
       if (!/matchme/.test(props[propName])) {
-        return new Error('Validation failed!');
+        return new Error(
+          `Invalid prop \`${propName}\` supplied to` +
```

Comment:
While we started using ES6 in the docs, I think we try to call it out explicitly when we do this (e.g. when we use classes and arrows). In this particular case I think we’re better off concatenating strings the traditional way so people who aren’t entirely comfortable with ES6 don’t get distracted by this.